x-pack/examples/triggers_actions_ui_example/public/components/page.tsx

```diff
@@ -49,18 +40,13 @@ export const Page: React.FC<PageProps> = (props) => {
 
   return (
     <EuiPageBody>
-      <EuiPageHeader>
-        <EuiPageHeaderSection>
-          <EuiTitle size="l">
-            <h1>{title}</h1>
-          </EuiTitle>
-        </EuiPageHeaderSection>
-      </EuiPageHeader>
-      <EuiBreadcrumbs responsive={false} breadcrumbs={breadcrumbs} />
-      <EuiSpacer />
-      <EuiPageContent>
-        <EuiPageContentBody>{children}</EuiPageContentBody>
-      </EuiPageContent>
+      <EuiPageTemplate.Header>
```

Comment:
Shouldn't we surround the `EuiPageTemplate.*` with `EuiPageTemplate`?

Reply:
I thought it just looked worse 😄 and since this is internal I didn't worry too much.

Reply:
How about using `EuiPageHeader` and `EuiPageSection ` instead (not the depreacted ones)?

Reply:
I'll see how it looks I played around with those too but now I am not sure if it was here or in `alerting_example`.